src/Skia/Perspex.Skia.iOS/SkiaView.cs

```diff
@@ -15,40 +15,47 @@
 
 namespace Perspex.Skia.iOS
 {
-    public abstract class SkiaView : GLKView
+    public abstract class SkiaView : UIView //GLKView
     {
-        [DllImport("__Internal")]
-        static extern IntPtr GetPerspexEAGLContext();
+        //[DllImport("__Internal")]
+        //static extern IntPtr GetPerspexEAGLContext();
 
         bool _drawQueued;
-        CADisplayLink _link;
+        //CADisplayLink _link;
         static EAGLContext GetContext()
         {
-            //Ensure initialization
-            MethodTable.Instance.SetOption((MethodTable.Option)0x10009999, IntPtr.Zero);
-            var ctx = GetPerspexEAGLContext();
-            var rv = Runtime.GetNSObject<EAGLContext>(ctx);
-            rv.DangerousRetain();
-            return rv;
+            /* No longer needed with SkiaSharp, but require a variant for HW accel
+                //Ensure initialization
+                MethodTable.Instance.SetOption((MethodTable.Option)0x10009999, IntPtr.Zero);
+                var ctx = GetPerspexEAGLContext();
+                var rv = Runtime.GetNSObject<EAGLContext>(ctx);
+                rv.DangerousRetain();
+                return rv;
+            */
```

Comment:
It may be useful to retain commented-out code such as this *iif* this code will be relevant to how this will be done in skiasharp. Will/is the skiasharp API for this be similar?